packages/components/src/view/component.tsx

```diff
@@ -30,7 +30,8 @@ import type { ViewProps } from './types';
 // @ts-expect-error
 export const View: WordPressComponent<
 	'div',
-	ViewProps & RefAttributes< any >
+	ViewProps & RefAttributes< any >,
+	true
```

Comment:
temporary fix so that I could see other type errors. (This will be solved by https://github.com/WordPress/gutenberg/pull/49960)